app/jobs/education_form/create_daily_spool_files.rb

```diff
@@ -91,25 +91,37 @@ def format_application(form)
 
     private
 
-    # Used in the INIT header of the 22-1990 template.
+    # If multiple benefit types are selected, we've been told to just include whichever
+    # one is 'first' in the header.
     def form_type(application)
-      {
-        CH33:       'CH33',
-        CH33_30:    'CH33',
-        CH33_1606:  'CH33',
-        CH33_1607:  'CH33',
-        CH1606:     'CH1606',
-        CH30:       'CH30',
-        CH32:       'CH32',
-        NA:         'NA'
-      }[application.form.to_sym]
+      return 'CH1606' if application.chapter1606
+      return 'CH33' if application.chapter33
+      return 'CH30' if application.chapter30
+      return 'CH32' if application.chapter32
     end
 
-    # Some descriptive text that's included near the top of the 22-1990 form
-    def disclosure(application)
-      contents = File.read(Rails.root.join('app', 'jobs', 'education_form', 'templates', "_#{application.form}.erb"))
-      ERB.new(contents).result(binding)
+    # Some descriptive text that's included near the top of the 22-1990 form. Because they can make
+    # multiple selections, we have to add all the selected ones.
+    # rubocop:disable Metrics/CyclomaticComplexity
+    def disclosures(application)
+      disclosure_texts = []
+      disclosure_texts << disclosure_for('CH30') if application.chapter30
+      disclosure_texts << disclosure_for('CH1606') if application.chapter1606
+      disclosure_texts << disclosure_for('CH32') if application.chapter32
+      if application.chapter33
+        ch33_type =
+          case application.benefitsRelinquished
```

Comment:
You may be able to get rid of the rubocop disabling by changing this to use a hash constant instead
```ruby
CH33_TYPES = {
  'chapter1607' => 'CH33_1607',
  ..
}

ch33_disclosure = CH33_TYPES[application.benefitsRelinquished] || 'CH33'
```